### DISCUSSION TITLE
Is it possible to download a font that does only contain the icon sets?

### DISCUSSION THREAD

**Discussion by mainrs** [ryanoasis/nerd-fonts#769] [Q&A] [answered]
That would allow me to use any font as a fallback without having to use the specific patched version of the font. Some fonts that I use are simply not patched or had to be bought.

**Comment by Finii** [comment, marked as answer]
There is https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/NerdFontsSymbolsOnly

Or follow #668

Note also #670
